.gitmodules

```diff
@@ -72,3 +72,6 @@
 [submodule "src/utf8proc"]
 	path = src/utf8proc
 	url = https://github.com/JuliaStrings/utf8proc
+[submodule "src/jaegertracing/opentelemetry-cpp"]
```

Comment:
the commit message is not accurate. it includes the items not covered by this change.

Reply:
also, could you please include `opentelemetry-cpp submodule` in the commit message as proposed by https://jenkins.ceph.com/job/ceph-pr-submodules/67495/console ?